client/src/cardComponent/cardComponent.js

```diff
@@ -0,0 +1,59 @@
+import React from "react"
+import { Carousel, Card } from "react-bootstrap"
+import "./cardComponent.css"
+function cardComponent(props) {
+  return (
+    <div className="Society-cards">
+      <h1 className="heading">Societies under Geeks</h1>
+    <Carousel>
+      <Carousel.Item>
+        <Card style={{ width: '18rem' }}>
+          <Card.Body>
+            <Card.Title>Card Title</Card.Title>
+            <Card.Subtitle className="mb-2 text-muted">Card Subtitle</Card.Subtitle>
+            <Card.Text>
+              Some quick example text to build on the card title and make up the bulk of
+              the card's content.
+    </Card.Text>
+            <Card.Link href="#">Card Link</Card.Link>
+            <Card.Link href="#">Another Link</Card.Link>
```

Comment:
You can avoid using `href="#"` as it might produce warning in the browser , you can replace it with href="https://" or provide a valid, navigable address as the href value